### DISCUSSION TITLE
Pictures copy paste from Word doesn't work anymore

### DISCUSSION THREAD

**Discussion by grandixximo** [requarks/wiki#3727] [Error / Bug Report]
**Describe the bug**
On version 2.0.12 i was able to copy content with pictures in the WYSIWYG editor, and save content, and everything would work, on version 2.1.113 every page that has a picture that i imported this way cannot be saved, instead i get this error

`Network error: JSON.parse: unexpected character at line 1 column 1 of the JSON data`

Also any new page that i try to create with a picture copied from word gives the same error

**To Reproduce**
1. Make a file in Word with images
2. Copy the content of the file in a new WYSIWYG page 
3. Save

**Expected behavior**
Pages can be saved normally like on version 2.0.12

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04
 - Wiki.js version: 2.1.113
 - Database engine: PostgreSQL 10.10

**Comment by andris-grishan** [comment]
We have the same issue.

OS: Ubuntu 18.04
Wiki.js version: 2.1.113
Database engine: PostgreSQL 10.12

**Comment by snatch2013** [comment]
+ the same

**Comment by mlafrance-complianceds** [comment]
We are experiencing the same

**Comment by grandixximo** [comment]
Any plans on a fix for this?

**Comment by BitWiseSE** [comment]
I had put an earlier comment about MS-Word images and how to use copy/paste to get them into WikiJS.

After further experimenting, I've come to this realization, which may be of use to some:

1. Your word document should be structured, and the Heading 1 and Heading 2 should be called 'Heading 1' and 'Heading 2'.
2. Your images should be "in-line" wrapping.

Using those two simples "rules", my experience is that the whole document comes through nicely using copy/paste.
